pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSinks.java

```diff
@@ -168,6 +168,8 @@ public void runCmd() throws Exception {
             localRunArgs.add(new Gson().toJson(sinkConfig));
             for (Field field : this.getClass().getDeclaredFields()) {
                 if (field.getName().startsWith("DEPRECATED")) continue;
+                if(field.getName().startsWith("this$0")) continue;
+                if(field.getName().startsWith("$")) continue;
```

Comment:
I think it should be `contains()` instead of `startsWith()`, no? Anything with `$` was auto created by compiler anyway.

Reply:
Good point. I have consolidated the logic